Source/WebGPU/WebGPU/CommandEncoder.mm

```diff
@@ -1731,6 +1732,7 @@ static bool areCopyCompatible(WGPUTextureFormat format1, WGPUTextureFormat forma
     return nil;
 }
 
+#if ! ENABLE(WEBGPU_SWIFT)
```

Comment:
I think it's fine but the space between the `!` and `ENABLE` seems unintentional perhaps?